indico/MaKaC/services/implementation/registration.py

```diff
@@ -77,11 +78,9 @@ def _handleGet(self):
 class ConferenceEticketQRCode(RegistrationModifBase):
 
     def _getAnswer(self):
-
-        consumers = dict((consumer.getName(), consumer) for consumer in ConsumerHolder().getList())
-
-        if "indico-checkin" not in consumers:
-            raise NoReportError(_("There is no indico-checkin consumer key for OAuth"))
+        oauth_checkin = OAuthApplication.find_first(name='Indico Checkin')
```

Comment:
Yeah, this is :cancer:. I'd go for the ID as well, but we are not displaying it in the interface.